commontype/fee_config.go

```diff
@@ -0,0 +1,61 @@
+package commontype
+
+import "math/big"
+
+// FeeConfig specifies the parameters for the dynamic fee algorithm, which determines the gas limit, base fee, and block gas cost of blocks
+// on the network.
+//
+// The dynamic fee algorithm simply increases fees when the network is operating at a utilization level above the target and decreases fees
+// when the network is operating at a utilization level below the target.
+// This struct is used by params.Config and precompile.FeeConfigManager
+// any modification of this struct has direct affect on the precompiled contract
+// and changes should be carefully handled in the precompiled contract code.
+type FeeConfig struct {
```

Comment:
did we need to move this to a separate package to avoid a circular dependency? I'm not a big fan of having a commonytype package with just this in it.

Reply:
yes :/. I've tried other packages but no luck :/